hat/assets/js/apps/Iaso/components/LegendBuilder/utils.ts

```diff
@@ -32,16 +32,22 @@ export const getRangeValues = (
 };
 export const getThresHoldLabels = (
     scaleThreshold?: ScaleThreshold,
+    unit?: string,
```

Comment:
```suggestion
    unit?: string = "%",
```

Otherwise I think it will break existing legends that use `"%"` as unit